feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -204,15 +213,14 @@ func TestBasic(t *testing.T) {
 	if err != nil {
 		t.Fatalf("No IS-IS adjacency formed: %v", err)
 	}
-	// Allow 1s of lag between adjacency appearing and all data being populated
+	// Allow 1 Minute of lag between adjacency appearing and all data being populated
```

Comment:
same thing.

Reply:
It is calculated based on number of rpcs in this block of code